### PR TITLE
fix: add new nuxt configs

### PR DESCRIPTION
**Pull request by ankur-arch**

### CODE REVIEW

**Review by jharrell — DISMISSED** [review]

File: content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/900-prisma-nuxt-module.mdx [hunk 1]
```diff
@@ -355,4 +345,8 @@ export default defineNuxtConfig({
 })
 ```
 
-This configuration ensures that the module specifier is correctly mapped to the appropriate file.
\ No newline at end of file
+This configuration ensures that the module specifier is correctly mapped to the appropriate file.
+
+### Support on Bun
+
+The module is designed to support popular package managers such as npm, Yarn, and pnpm. However, as of `v1.4`, it does not fully support Bun due to an issue where the installation process loops indefinitely.
```
Comment (ankur-arch) [inline content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/900-prisma-nuxt-module.mdx]:
```suggestion
### Limitations in package manager support

The module is designed to work with popular package managers, including npm, Yarn, and pnpm. However, as of `v1.4`, it is not fully compatible with Bun due to an issue causing an indefinite installation loop.

Additionally, this package has not been tested with Deno and is therefore not officially **supported.**
```

File: content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/900-prisma-nuxt-module.mdx [hunk 2]
```diff
@@ -355,4 +345,10 @@ export default defineNuxtConfig({
 })
 ```
 
-This configuration ensures that the module specifier is correctly mapped to the appropriate file.
\ No newline at end of file
+This configuration ensures that the module specifier is correctly mapped to the appropriate file.
+
+### Limitations in package manager support
+
+The module is designed to work with popular package managers, including npm, Yarn, and pnpm. However, as of `v1.4`, it is not fully compatible with Bun due to an issue causing an indefinite installation loop.
```
Comment (ankur-arch) [inline content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/900-prisma-nuxt-module.mdx]:
```suggestion
The module is designed to work with popular package managers, including npm, Yarn, and pnpm. However, as of `v2.0`, it is not fully compatible with Bun due to an issue causing an indefinite installation loop.
```

**Comment by ankur-arch** [comment]
Hey @Aguix 👋,

Apologies for the delay! I’ve worked on a [PR](https://github.com/prisma/nuxt-prisma/pull/71) to support Nuxt layers, which has now been merged. The new `v2.0` release introduces support for a layered-based architecture with Nuxt! 

However, the docs [PR](https://github.com/prisma/docs/pull/6557) still awaiting an internal review and then merging.

> How do I create the `.nuxt` folder?

To generate and build the project, refer to the [GitHub Action workflow](https://github.com/prisma/nuxt-prisma/blob/main/.github/workflows/publish.yml#L29). 

I’ll also be working on creating a more detailed CONTRIBUTION guide for this soon.

File: content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/900-prisma-nuxt-module.mdx [hunk 3]
```diff
@@ -355,4 +345,10 @@ export default defineNuxtConfig({
 })
 ```
 
-This configuration ensures that the module specifier is correctly mapped to the appropriate file.
\ No newline at end of file
+This configuration ensures that the module specifier is correctly mapped to the appropriate file.
+
+### Limitations in package manager support
+
+The module is designed to work with popular package managers, including npm, Yarn, and pnpm. However, as of `v2.0`, it is not fully compatible with Bun due to an issue causing an indefinite installation loop.
```
Comment (ankur-arch) [inline content/200-orm/800-more/600-help-and-troubleshooting/100-help-articles/900-prisma-nuxt-module.mdx]:
```suggestion
The module is designed to work with popular package managers, including npm, Yarn, and pnpm. However, as of `v0.2`, it is not fully compatible with Bun due to an issue causing an indefinite installation loop.
```

**Review by jharrell — APPROVED** [review]

**Pull request merged**